zipline/data/data_portal.py

```diff
@@ -916,6 +916,8 @@ def _get_minute_window_for_equities(
             return np.full((len(minutes_for_window), len(assets), np.nan))
 
         num_minutes = len(minutes_for_window)
+        start_date = normalize_date(minutes_for_window[0])
```

Comment:
Two thoughts

1) We could have the tradesim loop set the current_day on the dataportal, like we used to, to avoid this call to `normalize_date`, but that probably isn't worth it.
2) Anytime we use daily history, we know that it's going to call this method with the minute from the last day - is it worth passing in an extra param to signal that we know it's all on the same day, so no need to `normalize_date` twice and then do a date comparison?